slick/src/main/scala/slick/jdbc/JdbcDataSource.scala

```diff
@@ -150,6 +150,7 @@ object HikariCPJdbcDataSource extends JdbcDataSourceFactory {
 
     // Pool configuration
     hconf.setConnectionTimeout(c.getMillisecondsOr("connectionTimeout", 1000))
+    hconf.setValidationTimeout(c.getMillisecondsOr("validationTimeout", 500))
```

Comment:
The new parameter should be document in the scaladoc for JdbcBackend.Database.forConfig